xarray/core/variable.py

```diff
@@ -1993,23 +1993,34 @@ def rolling_window(
             ),
         )
 
-    def coarsen(self, windows, func, boundary="exact", side="left", **kwargs):
+    def coarsen(self, windows, func, boundary="exact", side="left", keep_attrs=False, **kwargs):
         """
         Apply reduction function.
         """
         windows = {k: v for k, v in windows.items() if k in self.dims}
         if not windows:
             return self.copy()
 
+        if "keep_attrs" in kwargs:
```

Comment:
`keep_attrs` isn't in kwargs any more. After that it *should* work =)

Reply:
Good catch! It is passing tests locally now!